tools/cpp/windows_cc_configure.bzl

```diff
@@ -98,6 +98,8 @@ def find_vc_path(repository_ctx):
     _auto_configure_warning_maybe(repository_ctx, "Looking for VS%VERSION%COMNTOOLS environment variables, " +
                                                   "eg. VS140COMNTOOLS")
     for vscommontools_env in [
+        "VS160COMNTOOLS",
+        "VS150COMNTOOLS",
```

Comment:
I suspect the current logic doesn't work for `VS150COMNTOOLS` and probably not for `VS160COMNTOOLS` (havn't checked though, as I don't have MSVS 2019 installed).

On my VM, for MSVS 2017, where `VS150COMNTOOLS` is set, the file: "vcvarsqueryregistry.bat" does not exist in this directory. See patches that I tried to apply in context of this issue: #7821. 

In my patch, I tried to source "VsDevCmd.bat", that is also used in LibreOffice project, see: [1].

* [1] https://cgit.freedesktop.org/libreoffice/core/tree/configure.ac#n3234

Reply:
Thanks! Updated, PTAL.

Reply:
Thanks. LGTM.